test/ClangImporter/objc_async.swift

```diff
@@ -43,6 +43,8 @@ func testSlowServer(slowServer: SlowServer) async throws {
   let _: Int = await slowServer.bestName("hello")
   let _: Int = await slowServer.customize("hello")
 
+  slowServer.unavailableMethod() // expected-warning{{'unavailableMethod' is unavailable from asynchronous contexts}}
```

Comment:
This Just Worked without any code changes? Awesome! Thanks for checking it.

Reply:
Yep, all just workd.

Reply:
Yes, it all just worked.